assets/vue/views/course/Create.vue

```diff
@@ -31,18 +30,17 @@ import courseService from "../../services/courseService"
 import { useI18n } from "vue-i18n"
 import { useNotification } from "../../composables/notification"
 
-const store = useStore()
 const item = ref({})
 const router = useRouter()
 const { t } = useI18n()
 
-const isLoading = computed(() => store.getters['course/getField']('isLoading'))
-const violations = computed(() => store.getters['course/getField']('violations'))
-const courseData = ref({})
+const isLoading = ref(false)
+const violations = ref(null)
```

Comment:
The reference violations is used in the component CourseForm but this component has no property errors...
```html
<CourseForm
      ref="createForm"
      :errors="violations"
      :values="item"
      @submit="submitCourse"
    />
```